dockerfiles/run/guice/cassandra/destination/conf/mailetcontainer.xml

```diff
@@ -70,7 +70,13 @@
                 <notice>550 - Requested action not taken: no such user here</notice>
             </mailet>
 
-            <mailet match="SMTPAuthSuccessful" class="ToProcessor">
+            <mailet match="SMTPAuthSuccessful" class="SetMailAttribute">
```

Comment:
![screenshot from 2017-05-10 14-00-23](https://cloud.githubusercontent.com/assets/6928740/25886704/0a6d20e8-3589-11e7-8a0d-7279334ee14b.png)

```
            <mailet match="HasMailAttribute=org.apache.james.jmap.send.MailMetaData.messageId" class="ToProcessor">
                <processor>relay</processor>
            </mailet>
```

Taken from this exact same file, on master <3